extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -146,6 +153,19 @@ export class DraggableDrawer extends AMP.BaseElement {
       )
     );
 
+    if (
+      isPageAttachmentUiV2ExperimentOn(this.win) &&
+      this.element.tagName === 'AMP-STORY-PAGE-ATTACHMENT'
```

Comment:
If it's for only one client of this abstraction then the code shouldn't live here.
Let's decide on if this change is for all draggable drawers or just page attachments, and write the code in the appropriate class.